Doc/glossary.rst

```diff
@@ -299,6 +299,11 @@ Glossary
       with :term:`abstract base classes <abstract base class>`.)  Instead, it
       typically employs :func:`hasattr` tests or :term:`EAFP` programming.
 
+   dunder
+      A dunder is a variable, method or package whose name is with two leading
```

Comment:
We should say that it's short for "double underscore".

Reply:
```
dunder
   A dunder is a variable, method or package whose name is with two double underscore
```
?

Reply:
or just `dunder = double underscore" ?